pkg/controller/installation/products/ups/reconciler.go

```diff
@@ -0,0 +1,159 @@
+package ups
+
+import (
+	"context"
+
+	upsv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/push/v1alpha1"
+	v1 "github.com/openshift/api/route/v1"
```

Comment:
I would name it `routev1` to keep it consistent with other imports (metav1, apssv1...)